README.md

```diff
@@ -378,6 +378,59 @@ the ownership voucher and ownership certificate.
     we have validated the identity and integrity of the device and its
     software components. It is ready to serve traffic.
 
+### A Note on How To Secure The First gNSI.Certz Connection After Successful Attestation
+
+In some deployment environment, an user might prefer to install production CA
+issued cert ONLY if the device has passed the attestation. iDevID cert is a good
+choice for securing the very first gNSI.Certz connection that installs the
+production issued cert. However, what's a simple and vendor neutral way to tell
+the device to use iDevID cert credentials for the first gNSI.Certz connection?
+
+The idea is that if
+[a SSL profile](https://github.com/openconfig/gnsi/blob/d5abc2e8fa51d7b57b49511655b71422638ce8cf/certz/certz.proto#L291-L292)
+includes a cert (e.g. iDevID) but not a private key, the device should check if
+there is any private key available on the device (e.g. private key in the TPM)
+matches the provided cert. If no existing private key is available for the SSL
+profile, the configs should be rejected. Applying the idea, the user journey
+would be as the following:
+
+1.  Upon establishing the Bootz gRPC connection, bootstrap server extracts the
```

Comment:
The linter doesn't seem to like the way this list is formatted. Could you take a look?